tests/framework/extensions/clusters/clusterconfig.go

```diff
@@ -21,7 +21,8 @@ type ClusterConfig struct {
 	AgentEnvVarsRKE1     *[]management.EnvVar                     `json:"agentEnvVarsRKE1" yaml:"agentEnvVarsRKE1"`
 	ClusterAgent         *management.AgentDeploymentCustomization `json:"clusterAgent" yaml:"clusterAgent"`
 	FleetAgent           *management.AgentDeploymentCustomization `json:"fleetAgent" yaml:"fleetAgent"`
-	Etcd                 *rkev1.ETCD                              `json:"etcd" yaml:"etcd"`
+	ETCD                 *provisioningInput.ETCD                  `json:"etcd" yaml:"etcd"`
```

Comment:
Why are you creating an etcd config instead of just using the one from rkev1?

Reply:
When we're provisioning a cluster, we need to be able to specify from our config what we want to use. This is part of broader test cases we plan to have down the line.

Reply:
Understood, what I'm talking about is why not rkev1.ETCD struct instead of rewriting it?

Reply:
Spoke offline, addressed this.